src/common/pages/submit.tsx

```diff
@@ -826,7 +826,7 @@ class SubmitPage extends BaseComponent<Props, State> {
 
   buildMetadata = () => {
     const { tags, body, description, selectedThumbnail, selectionTouched } = this.state;
-    const meta = extractMetaData(body);
+    const { ["thumbnails"]: thumbnails, ...meta } = extractMetaData(body);
```

Comment:
@igormuba 
`const { thumbnails, ...meta} = extractMetaData(body)` should work as well, right?

Reply:
Yes, both have the same effect. Just to be sure I tried your way and it worked so I changed it, looks cleaner than my original solution, has the same effect